File: Bukkit/src/main/java/me/mrkirby153/kcutils/scoreboard/items/ElementText.java
package me.mrkirby153.kcutils.scoreboard.items;

import me.mrkirby153.kcutils.scoreboard.ScoreboardElement;

public class ElementText implements ScoreboardElement {

    private String text;

    public ElementText(String text) {
        if(text.length() < 16){
            for (int i = 0; i < 15 - text.length(); i++) {
                text += " ";
            }
        }
        this.text = text;
    }

    @Override
    public String[] getLines() {
        String[] text = new String[1];
        text[0] = this.text.substring(0, Math.min(this.text.length(), 40));
        return text;
    }
}

File: Bukkit/src/main/kotlin/me/mrkirby153/kcutils/scoreboard/items/ElementText.kt
package me.mrkirby153.kcutils.scoreboard.items

import me.mrkirby153.kcutils.scoreboard.ScoreboardElement

class ElementText(text: String) : ScoreboardElement {

    private val text: String

    init {
        var t = text
        if (t.length < 16) {
            for (i in 0 until 15 - t.length) {
                t += " "
            }
        }
        this.text = t
    }

    override val lines: Array<String>
        get() {
            val text = arrayOf<String>()
            text[0] = this.text.substring(0, Math.min(this.text.length, 40))
            return text
        }
}